internal/docs/readme.go

```diff
@@ -204,6 +209,9 @@ func renderReadme(fileName, packageRoot, templatePath string, linksMap linkMap)
 			}
 			return linksMap.RenderLink(args[0], options)
 		},
+		"header": func(args ...string) (string, error) {
```

Comment:
This depends on the user leaving ``{{header}}` in the readme in order to add the "Do not modify" string to the generated files.

I could also inject the string into all generated READMEs, regardless of if this {{header}} is in it, but I don't know if this would cause problems with other package types, so I didn't do that. If it won't cause problems, I could change to add to all generated files.

Reply:
Yep, it could make sense to inject it in all generated files, but we would need to check that validation doesn't fail for packages that don't include this header.